infra/modules/azure_core_infra/variables.tf

```diff
@@ -59,3 +59,10 @@ variable "nat_enabled" {
   description = "Flag to enable nat gateway creation"
   default     = false
 }
+
+
+variable "gh_runner_snet" {
+  type        = string
+  default     = "10.0.242.0/23"
```

Comment:
it is the same value that we have used in other contexts, and in any case if the module is instantiated without configuring any subnet it is a value that does not affect any other IP range and leaves all the initial ones free

Reply:
I'd clarify with a comment the origin of this value. If it's a random value we should say it is random. If we have constraints we should clarify which ones.

Reply:
It was the first IP range available in IO WEU VNet at the time.

As we are providing this module for isolated and ephemeral test environments, it might be good to have default values for IP blocks, to accelerate things

Reply:
What do you propose as default?

Reply:
The ones added in this PR are ok to me